### ISSUE TITLE
Insufficient color contrast on links for page numbers

### ISSUE THREAD

**Issue by yuktea** [openfoodfacts/openfoodfacts-server#7265]
### Description of the issue

This issue was reported by the Accessibility Foundation (https://github.com/openfoodfacts/openfoodfacts-server/issues/4640)
Quoting from [accessibility_report.pdf](https://github.com/openfoodfacts/openfoodfacts-server/files/5702582/Report.OpenFoodFact.15-12-2020.pdf)
Quoting this report:
1.4.3 Contrast (minimum)

> On the homepage (https://nl.openfoodfacts.org/) you will find the buttons under the 
> heading "Last added products" with the numbers 2, 3 and etc. The buttons are light gray 
> compared to the white background; the contrast ratio is 2.6: 1 which should be at least 
> 4.5:1 for people with a visual impairment.

![image](https://user-images.githubusercontent.com/73362292/187096296-c8b400d5-91a5-4650-a544-433d17a66226.png)

**Comment by MaruthiKo** [comment]
Hi, I would like to work on this issue.

**Comment by MaruthiKo** [comment]
Alright Done will, look into it

**Comment by aakankshabhende** [comment]
@alexgarel I would like to work on this issue

**Comment by yuktea** [comment]
@aakankshabhende you've just been assigned another issue, would you still like to work on it?
@MaruthiKo : are you working on it?

**Comment by MaruthiKo** [comment]
> @aakankshabhende you've just been assigned another issue, would you still like to work on it?
> @MaruthiKo : are you working on it?
 
Understood the changes but could not find the file to make changes

**Comment by alexgarel** [comment]
Use developper tool in you navigator to see which CSS rules are involved.
Eventually add one to customize.
As said above, it's in _off.scss

**Comment by MaruthiKo** [comment]
Yea did try that, it showed that the changes are in _pagination.scss
Will look into that again
Thanks for the help
